Src/ParachutesControl.c

```diff
@@ -45,17 +68,94 @@ int32_t readPressure(BarometerData* data)
     return (int32_t)pressure;
 }
 
-void filterSensors(int32_t current_accel, int32_t current_pressure, int32_t positionVector[3])
+
+
+/**
+ * Takes an old state vector and current state measurements and
+ * converts them into a prediction of the rocket's current state.
+ *
+ * Params:
+ *   oldState - (KalmanStateVector) Past altitude, velocity and acceleration
+ *   currentAccel - (double) Measured acceleration
+ *   currentAltitude - (double) Measured altitude
+ *   dt - (double) Time since last step
+ *
+ * Returns:
+ *   newState - (KalmanStateVector) Current altitude, velocity and acceleration
+ */
+struct KalmanStateVector filterSensors(
+    struct KalmanStateVector oldState,
+    int32_t currentAccel,
+    int32_t currentPressure,
+    double dt
+)
 {
-    // TODO
-    positionVector[0] = 0;
-    positionVector[1] = 0;
-    positionVector[2] = 0;
+    struct KalmanStateVector newState;
+    
+    // TODO: Figure out what the conversion factor is.
```

Comment:
Hey, let's do this as if the 16G conversion was working as we expected. i.e. 2048 is 1G.